openlibrary/plugins/importapi/code.py

```diff
@@ -458,12 +458,19 @@ def POST(self):
 
     def error(self, reason):
         d = json.dumps({ "status" : "error", "reason" : reason})
-        return web.HTTPError("400 Bad Request", {"Content-Type": "application/json"}, d)
+        return web.HTTPError("400 Bad Request", {"content-type": "application/json"}, d)
 
 
     def auth_failed(self, reason):
         d = json.dumps({ "status" : "error", "reason" : reason})
-        return web.HTTPError("401 Authorization Required", {"WWW-Authenticate": 'Basic realm="http://openlibrary.org"', "Content-Type": "application/json"}, d)
+        return web.HTTPError(
+            "401 Authorization Required",
+            {
+                "www-authenticate": 'Basic realm="http://openlibrary.org"',
```

Comment:
In some places above we've changed `Basic` -> `basic`

There are several other locations here where we don't make that change

https://github.com/internetarchive/openlibrary/pull/5840/files#diff-fefad9036d5577fbc4e634459d3638bae0e47a93269b28302f90c660b276cc7bL37

Reply:
Nice catch!  Thanks.